automation/e2e-functests.setup.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -xeu
+
+# This script prepares an ephemeral cluster with
+# latest changes CNAO and latest commit KubeVirt on top.
+# And clones the desired component repository
+#
+# This script also exports:
+# $KUBECONFIG
+#  kubeconfig full path so you can use`kubectl` binary or
+#  CNAO's `./cluster/kubectl.sh` script directly.
+#
+# $TMP_PROJECT_PATH
+# Latest changes CNAO temp directory
+#
+# $TMP_COMPONENT_PATH
+# The desired component temp directory
+#
+# Example:
+# export COMPONENT="KubeMacPool"
+# source automation/e2e-functest.setup.sh
+# cd ${TMP_COMPONENT_PATH}
+# KUBECONFIG=$KUBECONFIG make functests
+
+source hack/components/git-utils.sh
+source hack/components/yaml-utils.sh
+
+source automation/check-patch.setup.sh
+
+teardown() {
+    #make cluster-down
+    # rm -rf "$kubevirt_temp_dir"
+    # rm -rf "$component_temp_dir"
+    echo hi
+}
+
+trap teardown EXIT SIGINT SIGTERM SIGSTOP
+
+export COMPONENT=${COMPONENT:-kubemacpool}
```

Comment:
It seems you moved some common stuff to `e2e-functests.setup.sh`, similar to `check-patch.setup.sh`, right?. 
If so, then it's common so please remove traces of kubemacpool from it.

Reply:
Will do